### DISCUSSION TITLE
[Weekly Curation Report]: November 27 - December 11

### DISCUSSION THREAD

**Discussion by mkutmon** [wikipathways/wikipathways-help#90] [curation-reports]
### Recent changes

 * 53 curation edits on multiple pathways by JuliaUM, Khanspers, Ulaganathan, Egonw, Conroy lipids, Ondrej Kuda, DeSl, Alexmadsen1
   * ontology tags
   * identifiers
   * improvements and extensions
* Updated one outdated curated tag

### Under construction

 * Nothing to do.

### New pathways

 * WP5428- "FGFR4 p.G388R signaling pathway" by Ulaganathan was uploaded three times - will propose to delete two of them but I will also reach out to the author to prevent future uploads (proposed to keep = WP5428, propose to delete = WP5429, WP5430)
 * WP5431 - "FLT3 p.Gln640SerfsTer8 Signaling Pathway" by Ulaganathan (added ontology tags + quality tag, fixed missing ref)
 * WP5432 - "Omega-3-fatty acids in senescence" by JuliaUM (added missing description tag)

### Deleted Pathways

 * Nothing to do.

### Additional notes

* Issue with the list of pathways that are requested to be deleted. If the pathways have the same name, only one of them seems to show up. 

@susancoort, you are up next.

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by mkutmon** [comment]
I didn't see a curation report for the week starting Nov 27, so I looked at the last two weeks.

**Comment by khanspers** [comment]
> ### New pathways
> * WP5428- "FGFR4 p.G388R signaling pathway" by Ulaganathan was uploaded three times - will propose to delete two of them but I will also reach out to the author to prevent future uploads (proposed to keep = WP5428, propose to delete = WP5429, WP5430)

@mkutmon : Actually the author, Ulaganathan, had requested deletion of WP5429 (and WP5427) on Nov 30 and I somehow only deleted the latter. I just now deleted WP5429. Still a good idea to contact them about WP5430 before deleting.
